.tool-versions

```diff
@@ -1 +1 @@
-nodejs 14.12.0
+nodejs 14.15.0
```

Comment:
Should this be on the major – like `15.0.1`?

Reply:
Done ✅